### ISSUE TITLE
about tf.nn.tanh

### ISSUE THREAD

**Issue by guvcolie** [zhxl0903/CSCD94_VESPCN#2]
Thank you for your excellent code! I have a question, could you help me?
You use tf.nn .tanh() activation to 't1_fine_warp' in function 'spatial_transformer', so the range should be (-1, 1). However, when you preprocess the dataset, you divide the image by 255.0, so I think the three frame's range is (0, 1). After warping, the current frame range is (0, 1), the warped_prev_frame range is (-1, 1), the warped_next_frame range is (-1, 1), you concat the three frame to be 'imgSet'. 
Do you think that's correct?
Thank you!

**Comment by zhxl0903** [comment]
Hi, thanks for the attention!

Yes, the activation range is (-1, 1). It will work for normalized images with range (0, 1) or (-1, 1).
Part of the loss function for spatial transformer enforces the warped frames' range to be same as the current frame, (0, 1).  So, all 3 frames' ranges will be (0, 1) after training. 

Thanks!
zhxl0903
